src/components/themed/Thermostat.tsx

```diff
@@ -7,13 +7,15 @@ import { EdgeText } from './EdgeText'
 
 interface Props {
   ratio: number
+  textColor?: string
```

Comment:
The only place thermostat is used always receives the textColor prop so this can be required